x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/services/openai/embeddings/OpenAiEmbeddingsServiceSettingsTests.java

```diff
@@ -0,0 +1,381 @@
+/*
```

Comment:
Sigh, these should have been moved. There mostly the same as before but I did add a few test cases around the dimensions behavior.